tests/integration.js

```diff
@@ -9,6 +9,8 @@ require('../lib/impress');
 
 ncp.limit = 16;
 
+//let timer = setTimeout(taskExit, 5000);
```

Comment:
FWIW, I've experienced occasional integration test hanging for a long time. Right now it works for me both in master and in this PR.